### PR TITLE
Flask boost

### PR DESCRIPTION
**Pull request by fentontaylor**
### PR Includes:
- Adds packages gunicorn and gevent for improved multi-thread performance
- Changes import structure so server runs properly with gunicorn
- Adds patched.py to monkey patch available functions with gevent
- Updates README with new results

### CODE REVIEW

File: flask/patched.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from gevent import monkey
+monkey.patch_all()
+
+from app import app
```
Comment (tylorschafer) [inline flask/patched.py]: This is cool! What does it do?

**Comment by tylorschafer** [comment]
Amazing how such a small change can boost performance by this much!

File: flask/patched.py (hunk 1 above)
Comment (fentontaylor) [reply flask/patched.py]: It intercepts a bunch of common python functions and makes them able to run asynchronously. Magic basically. I'm not 100% sure if it is intercepting everything we want, but I'll look into it more later.

Comment (tylorschafer) [reply flask/patched.py]: That's super cool. Nice Work. Merging

**Pull request merged**